code/modules/medical/cloner_defects.dm

```diff
@@ -238,36 +258,56 @@ ABSTRACT_TYPE(/datum/cloner_defect/missing_organ)
 		qdel(src.owner.organHolder.organ_list[data["organ_string"]])
 
 /datum/cloner_defect/missing_organ/minor
-	severity = CLONER_DEFECT_SEVERITY_MINOR
+	severity = CLONER_DEFECT_SEVERITY_MAJOR
 
 	init()
-		src.data = list("organ_string" = pick("tail", "left_kidney", "right_kidney", "stomach", "intestines", "appendix"))
+		src.data = list("organ_string" = pick("tail", "left_kidney", "right_kidney", "stomach", "intestines", "appendix", "liver",))
 
 /datum/cloner_defect/missing_organ/major
 	severity = CLONER_DEFECT_SEVERITY_MAJOR
 
 	init()
-		src.data = list("organ_string" = pick("liver", "left_eye", "right_eye", "left_lung", "right_lung")) // no skull, head, brain; instant death sucks. also removed heart
+		src.data = list("organ_string" = pick("left_eye", "right_eye", "left_lung", "right_lung")) // no skull, head, brain; instant death sucks. also removed heart
+
+/datum/cloner_defect/missing_organ/heart
+	severity = CLONER_DEFECT_SEVERITY_MAJOR
+	weight = 10
+
+	init()
+		src.data = list("organ_string" = pick("heart")) // this happens rarely, and allows some chance to still be saved
+
+proc/filter_muteraces(var/type)
+	var/datum/mutantrace/muteman = type
+	return !initial(muteman.dna_mutagen_banned)
+
+proc/filter_rare_muteraces(var/type)
+	var/datum/mutantrace/muteman = type
+	return initial(muteman.allow_rare_defect) && !initial(muteman.dna_mutagen_banned)
 
 /// Set to a random (safe) mutantrace after cloning
 /datum/cloner_defect/random_mutantrace
 	name = "Unexpected Genetic Development"
 	desc = "Subject's DNA has mutated into that of a different species."
-	severity = CLONER_DEFECT_SEVERITY_MINOR
+	severity = CLONER_DEFECT_SEVERITY_MAJOR
 	stackable = FALSE
 	var/orig_mutantrace_type = null //! Holds their original mutantrace type so we can change them back if the defect is removed
 
 	init()
 		src.data = list("new_mutantrace_type" = null)
-		while (!data["new_mutantrace_type"] || initial(data["new_mutantrace_type"]:dna_mutagen_banned))
-			data["new_mutantrace_type"] = pick(concrete_typesof(/datum/mutantrace))
+		data["new_mutantrace_type"] = pick(filtered_concrete_typesof(/datum/mutantrace, /proc/filter_muteraces))
 
 	on_add()
 		. = ..()
 		src.orig_mutantrace_type = src.owner.mutantrace?.type
 		src.owner.set_mutantrace(data["new_mutantrace_type"])
 
-// TODO abstract status effect defects more
+	rare
+		severity = CLONER_DEFECT_SEVERITY_MAJOR
+		weight = 30
+
+		init()
+			src.data = list("new_mutantrace_type" = null)
+			data["new_mutantrace_type"] = pick(filtered_concrete_typesof(/datum/mutantrace, /proc/filter_rare_muteraces))
```

Comment:
vetoing all of this. The mutantraces selected here range from "literal aliens" to "ancient jank mutantrace that should never see the light of day again", and none of that should be popping out of the cloner when cloning a human